docs/DeveloperGuide.md

```diff
@@ -210,18 +211,16 @@ unlikely to have) - `*`
 | `* * *`  | user                                       | filter by car brand preference | find customers who prefer a specific car brand easily         |
 | `* * *`  | user                                       | filter by address   | find customers who live near a specific region
 | `* * *`  | user                                       | filter by name   | find customers whose name match most closely the phone number I remember
-| `* * *`  | user                                       | filter by Date Of Birth   | find customers whose DoB match most closely the phone number I remember
+| `* * *`  | user                                       | filter by Date Of Birth   | find customers whose Date of Birth match most closely the phone number I remember
 | `* * *`  | user                                       | filter by email   | find customers whose email match most closely the phone number I remember
-| `* * *`  | user                                       | filter by phone number   | find customers whose phone number match most closely the phone number I remember
+| `* * *`  | user                                       | filter by phone number   | find customers whose phone number match most closely the name I remember
```

Comment:
name?? should be phone number

Reply:
if i change to phone number

find customers whose phone number match most closely the phone number I remember
dont make sense